cli.md

```diff
@@ -53,9 +53,15 @@ If `--name` is not specified, CLI will look for an environment variable, e.g. `e
 * `t2 run <file>` copy the file and its dependencies into Tessel's RAM & run immediately. Use this during development of your device application.
   * `[--lan]` deploy over LAN connection
   * `[--usb]` deploy over USB connection
+  * `[--slim]` true by default, copy only files needed by the program to run
```

Comment:
I guess you wouldn't anymore since it's the default behavior but it's still in `t2-cli` so it may be worth documenting.